src/deck.js

```diff
@@ -0,0 +1,38 @@
+export default class Deck {
+  constructor(cards) {
+    this.cards = cards;
+  }
+
+  rankOfCardAt(index) {
+    if (this.cards[index]) {
+      return this.cards[index].rank
+    } else if (this.cards.length > 1) {
+      return this.cards[this.cards.length - 1].rank
+    }
+  }
+
+  highRankingCards() {
+    let highRank = []
+    this.cards.forEach(card => {
+      if (card.rank > 10) {
+        highRank.push(card)
+      }
+    });
+    return highRank
+  }
```

Comment:
This doesn't seem like an efficient way of doing a map but I couldn't find any syntax in JavaScript for a map function. Whenever I need to use a map I've been declaring an array and pushing to it in the if statement. Is there a better way of doing this?